web_responsive/static/src/less/form_view.less

```diff
@@ -1,14 +1,47 @@
 /* Copyright 2016 Ponto Suprimentos Ltda.
+   Copyright 2018 Alexandre Díaz
  * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
 
+@sheet-margin: @sheet-padding;
+@chatter-side-width: @screen-md / 2.5;
+
+// Sided Chatter
+@media (min-width: @screen-md) {
+    .o_chatter_position_sided {
+        .o_form_view {
+            display: flex;
+            height: 100%;
+
+            .o_form_sheet_bg {
+                border-right: 1px solid @table-border-color;
+                overflow: auto;
+                flex: 1 1 auto;
+            }
+
+            .o_chatter {
+                overflow: auto;
+                flex: 0 0 @chatter-side-width;
+
+                .o_chatter_topbar {
+                    height: auto;
+                    flex-wrap: wrap;
+                }
+            }
+            .oe_chatter:empty {
```

Comment:
You can put this inside the `.o_chatter` block and use `&:empty` instead.

Reply:
The element only have the class "oe_chatter" when it's empty ... perhaps best use oe_chatter for all?

Reply:
Oh, I didn't know that. Then maybe yes, we can use `oe_chatter` for both.